src/network/serverpackethandler.cpp

```diff
@@ -1866,13 +1866,17 @@ void Server::handleCommand_HaveMedia(NetworkPacket *pkt)
 
 void Server::handleCommand_UpdateClientInfo(NetworkPacket *pkt)
 {
-	ClientDynamicInfo info;
+	ClientDynamicInfo info{};
```

Comment:
relies on the implicit default for `touch_controls` (I think)

Reply:
Is this a problem and what is the solution? This kind of initialization is also used elsewhere for `ClientDynamicInfo`.

Reply:
I was thinking that explicitly initializing it to `false` would be better.

Reply:
Updated.